spec/checkin_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'yaml'
+require './lib/library.rb'
+
+person = YAML.load_file('./lib/person.yml')
+describe Library do
+
+    subject { described_class.new }
+
+    it 'allow individuals to return books' do
+      subject.return_book(1)
+      expect(person[0][:available]).to eq true
+    end
+
```

Comment:
Does this behavior reflect the behavior of your implementation code? I believe you are calling `person.books` - here you are just storing the book list inside `person`. You want your test code to actually test your implementation code. This doesn't do so.